README.md

```diff
@@ -510,23 +709,43 @@ open-attestation dns txt-record get --location example.openattestation.com --net
 
 ### Wallet
 
-Creating a wallet
+You can create, encrypt, decrypt, or provide a wallet using the commands below.
+
+#### Creating a wallet
+
+The `wallet create` command creates a new wallet.
+
+The following is a command example:
 
 ```bash
 open-attestation wallet create --of ./tmp
+```
+
+The response looks like:
 
+```
 ℹ  info      Creating a new wallet
 ? Wallet password [hidden]
 …  awaiting  Encrypting Wallet [====================] [100/100%]
 ℹ  info      Wallet with public address 0x6348a96D96D56665C2E9619f81355918779f3d9E successfully created. Find more details:
 ✔  success   Wallet successfully saved into /path/to/tmp
 ```
 
-Encrypting a wallet (see [below](#providing-the-wallet) to find out how to provide the key)
+#### Encrypting a wallet 
+
+Using the `wallet encrypt` command, you can encrypt a wallet.
+
+>**Note:** If you want to provide the wallet private key, see the ["Providing the wallet"](#providing-the-wallet) section.
```

Comment:
I noticed we link to this article in almost every command. 

1. Do you think there is a better way to do this? 
2. If not, the current line is `See more options [here](#providing-the-wallet).` which I think sounds a bit vague so can we rephrase it to something clearer? e.g. `For more ways to provide the wallet, see [here](#providing-the-wallet).`

Reply:
updated all